### PR TITLE
1715 simplified login UI nebula

### PR DESCRIPTION
**Pull request by Daniel-Echegaray**

### CODE REVIEW

File: Sources/AccountSDKIOSWeb/Simplified Login UI/SimplifiedLoginNebula.swift [hunk 1]
```diff
@@ -0,0 +1,397 @@
+import SwiftUI
+import UIKit
+
+public class SimplifiedLoginManager {
+    public func presentSimplifiedLogin(presentingViewController: UIViewController) {
```
Comment (Daniel-Echegaray) [inline Sources/AccountSDKIOSWeb/Simplified Login UI/SimplifiedLoginNebula.swift:5]: Thinking about having a function like this as the main access point of Simplfied Login.

SimplifiedLoginManager is probably not a good name. Maybe SimplifiedLoginUIFactory.
SDK consumers will need to inject a User instance on constructing this class aswell.

Comment (zamorastp) [reply Sources/AccountSDKIOSWeb/Simplified Login UI/SimplifiedLoginNebula.swift:5]: I wonder if this brings additional benefits to developers. What about delivering just VC and let them create it and add to their navigation flow?

File: Sources/AccountSDKIOSWeb/Simplified Login UI/SimplifiedLoginNebula.swift [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+                                                 attributes: attributes)
+                                                 
+        view.textAlignment = .center
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+    
+}
+
+// MARK: SwiftUI Preview TODO: Remove this
```
Comment (zamorastp) [inline Sources/AccountSDKIOSWeb/Simplified Login UI/SimplifiedLoginNebula.swift:373]: Nice hack :D

File: Projects/AccountSDKIOSWebExample/ExampleWeb/ContentView.swift [hunk 3]
```diff
@@ -190,6 +198,24 @@ struct ContentView: View {
...
 
+struct SimplifiedLoginSwiftUIView: View {
+    var body: some View {
+        SimplifiedLoginViewControllerRepresentable()
+    }
+}
+
+struct SimplifiedLoginViewControllerRepresentable: UIViewControllerRepresentable {
+    func makeUIViewController(context: Context) -> SimplifiedLoginViewController {
+        let s = SimplifiedLoginViewController()
```
Comment (zamorastp) [inline Projects/AccountSDKIOSWebExample/ExampleWeb/ContentView.swift:209]: return SimplifiedLoginViewController()

File: Sources/AccountSDKIOSWeb/Simplified Login UI/SimplifiedLoginNebula.swift (hunk 1 above)
Comment (zamorastp) [reply Sources/AccountSDKIOSWeb/Simplified Login UI/SimplifiedLoginNebula.swift:5]: Also I encourage to have SimplifiedLoginViewController with public methods in one file, and all other  accompanying views in different files. Just to make it simple and clear for devs

**Review by zamorastp — APPROVED** [review]

**Pull request merged**